src/commons/zimbra-standard-colors.ts

```diff
@@ -11,15 +11,15 @@ export type ZimbraColorType = {
 
 export const ZIMBRA_STANDARD_COLORS: Array<ZimbraColorType> = [
```

Comment:
Could this constant be renamed to `CARBONIO_STANDARD_COLORS`? Also in the type we can get rid of the _zimbra_ name